osu.Game.Rulesets.Osu/Mods/OsuModSpunOut.cs

```diff
@@ -45,7 +45,8 @@ private void onSpinnerUpdate(Drawable drawable)
             // for that reason using ElapsedFrameTime directly leads to fewer SPM with Half Time and more SPM with Double Time.
             // for spinners we want the real (wall clock) elapsed time; to achieve that, unapply the clock rate locally here.
             double rateIndependentElapsedTime = spinner.Clock.ElapsedFrameTime / spinner.Clock.Rate;
-            spinner.RotationTracker.AddRotation(MathUtils.RadiansToDegrees((float)rateIndependentElapsedTime * 0.03f));
+            float rotationSpeed = (float)(spinner.HitObject.SpinsRequired / (spinner.HitObject.Duration / 1.01));
```

Comment:
I think it's a bit weird how the 1.01 factor is applied to the duration rather than the speed. Would prefer to write

```suggestion
            float rotationSpeed = (float)(1.01 * spinner.HitObject.SpinsRequired / spinner.HitObject.Duration);
```

The result should be theoretically the same, but I feel the above is clearer.

Also an inline comment about what the 1.01 adjustment is doing would be a good idea.

Reply:
Totally agree, it looks much nicer and makes more sense to apply the factor to the resulting speed, not the duration. I've updated the PR with this and an inline comment.